x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -178,9 +178,10 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
       // categorization job will always use a count agg, so give it
       // to the job creator now
       const count = newJobCapsService.getAggById('count');
+      const highCount = newJobCapsService.getAggById('high_count');
       const rare = newJobCapsService.getAggById('rare');
       const eventRate = newJobCapsService.getFieldById(EVENT_RATE_FIELD_ID);
-      jobCreator.setDefaultDetectorProperties(count, rare, eventRate);
+      jobCreator.setDefaultDetectorProperties(count, highCount, rare, eventRate);
```

Comment:
When I clone a `high_count` categorization job, it correctly goes into the Categorization job wizard but is selecting the `Rare` card.

Reply:
Fixed in https://github.com/elastic/kibana/pull/174252/commits/5e2efb53fe89a287ff2fb2925cede9548be96931